test/DynamoCoreTests/DynamoModelTestBase.cs

```diff
@@ -130,10 +130,10 @@ protected void EmptyScheduler()
             }
         }
 
-        protected void OpenModel(string relativeFilePath)
+        protected void OpenModel(string relativeFilePath, bool forceManualExecutionMode = false)
```

Comment:
@mjkkirschner @QilongTang Can we do this? Or do we need to add a new method?

Reply:
@QilongTang I can remove this if you would rather just handle it as part of your task.

Reply:
I think this is the way to do it.

Reply:
I guess we need to consider that this might effect others who have written tests against the test base classes..... 

I guess it's safest to add a new method unless you want to test this or do some research on the API compatibility of adding default arguments to protected methods?

Reply:
That's good, can you run self CI just to make sure?

Reply:
For the record, this did pass the self CI, but I'm going to remove this change for now and just merge the `RunType` changes.